pymatgen/io/vasp/sets.py

```diff
@@ -1195,6 +1123,15 @@ def next_g_size(cur_g_size):
         return ng_vec, [ng_ * finer_g_scale for ng_ in ng_vec]
 
 
+# create VaspInputGenerator alias to follow atomate2 terminology
+VaspInputGenerator = VaspInputSet
+
+
+@deprecated(replacement=VaspInputSet, deadline=(2025, 12, 31))
```

Comment:
open to suggestions on the `DictSet` deprecation deadline here. it's currently set to 19.5 months